e2e/cypress/support/saml_commands.js

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('checkLoginPage', (settings = {}) => {
+    // * Check the title
+    cy.title().should('include', settings.siteName);
+
+    // * Check elements in the body
+    cy.get('#loginId').should('be.visible').and(($loginTextbox) => {
+        const placeholder = $loginTextbox[0].placeholder;
+        expect(placeholder).to.match(/Email/);
+        expect(placeholder).to.match(/Username/);
+    });
+    cy.get('#loginPassword').should('be.visible').and('have.attr', 'placeholder', 'Password');
+    cy.findByText('Sign in').should('be.visible');
+});
+
+Cypress.Commands.add('checkSignUpPageSection', () => {
+    cy.findByText('Your guest account has no channels assigned. Please contact an administrator.').should('be.visible');
+});
+
+Cypress.Commands.add('checkChatSideBar', (settings = {}) => {
```

Comment:
nit: We're not using "chat" as we'd wanted to be more of messaging platform 😄 . Maybe change to `checklLeftSideBar`?